pkgs/development/php-packages/psysh/default.nix

```diff
@@ -16,7 +16,18 @@ php.buildComposerProject (finalAttrs: {
     url = "https://github.com/bobthecow/psysh/releases/download/v${finalAttrs.version}/composer-v${finalAttrs.version}.lock";
     hash = "sha256-ur6mzla3uXeFL6aEHAPdpxGdvcgzOgTLW/CKPbNqeCg=";
   };
-  vendorHash = "";
+
+  composerRepository = {
+    preBuild = ''
+      setComposeRootVersion
+      composer config platform.php 7.4
+      composer require --no-update symfony/polyfill-iconv symfony/polyfill-mbstring
+      composer require --no-update --dev roave/security-advisories:dev-latest
```

Comment:
Actually, this is not reproducible, is it? The hash would change over time since the lockfile is included in the output.

Reply:
No, this is fully reproducible actually, tested with `--rebuild`.

Reply:
What if `roave/security-advisories` releases a new version?